apps/auxtel/values-summit.yaml

```diff
@@ -10,3 +10,6 @@ cscs:
   - atspectrograph
   - atheaderservice
   - pmd1
+spec:
+  source:
+    targetRevision: tickets/DM-47527
```

Comment:
do you really want to commit with the target being a ticket branch? I think probably not. Also, I think this is something you can edit easily on argo-cd